orchestrator/test_runner/src/airdrop_proposal.rs

```diff
@@ -134,30 +98,97 @@ async fn submit_and_pass_airdrop_proposal(
     .await
     .unwrap();
 
-    trace!("Gov proposal submitted with {:?}", res);
-    let res = contact.wait_for_tx(res, TOTAL_TIMEOUT).await.unwrap();
-    trace!("Gov proposal executed with {:?}", res);
     info!(
         "Submit and pass airdrop proposal: for {} to {} recipients for {} gas",
-        total_array(amounts),
+        total_array(&amounts),
         recipients.len(),
         res.gas_used
     );
 
     vote_yes_on_proposals(contact, keys, None).await;
+    wait_for_proposals_to_execute(contact).await;
+
+    let mut error = false;
+    // make sure everyone got their airdrop amount
+    for (key, amount) in recipients.iter().zip(amounts.iter()) {
+        let balances = contact
+            .get_balance(*key, STAKING_TOKEN.to_string())
+            .await
+            .unwrap();
+
+        assert!(balances.is_some());
+        match balances {
+            Some(balance) => {
+                let big_amount: Uint256 = (*amount).into();
+                // assert_eq!(balances.unwrap().amount, big_amount);
+                if balance.amount != big_amount {
```

Comment:
yes it will, but the chance of doing so randomly is the same as the change of private key collision, which is of course negligable or it wouldn't be practical at all.